src/sparseml/pytorch/sparsification/pruning/modifier_pruning_base.py

```diff
@@ -117,10 +117,16 @@ def __init__(
         log_types: Union[str, List[str]] = None,
         global_sparsity: bool = False,
         allow_reintroduction: bool = False,
+        explicit_kwargs: Optional[List[str]] = ["params"],
```

Comment:
cannot default to a mutable object, let's default to `None` instead. Behavior for `None` should be to not propagate any extra kwargs. users who need `params` propagated should pass that explicitly